pippy/ManualPipelineStage.py

```diff
@@ -301,7 +301,6 @@ def get_fwd_send_ops(self) -> List[dist.P2POp]:
             dist.P2POp(dist.isend, out, self.next_stage, self.group)
             for out in output_tuples
         ]
-        raise AssertionError("invalid fwd_outputs type, cannot send")
```

Comment:
@H-Huang We will never hit here, right?

Reply:
yeah this is not necessary, thanks for removing